### ISSUE TITLE
potree 1.7 loading error

### ISSUE THREAD

**Issue by tjhanson** [potree/potree#1003]
I processed this point cloud using potree converter 2.0 and its using potree version 1.7.1.
https://fullcirclethinking.com/BarringtonPlaza/C_Plaza.html
On my computer everything loads fine, but on another computer the point cloud doesnt load. Both computers running windows, chrome and firefox same problem. There is no error in the console, and the point cloud shows up in scene objects, but its not visible. The odd thing is another point cloud processed using converter 1.6 loads in fine. What could be going wrong?
![image](https://user-images.githubusercontent.com/19607828/111678510-9ad36700-87dd-11eb-82a9-f3b3850dbe0a.png)
what shows up when point cloud doesnt load
![image](https://user-images.githubusercontent.com/19607828/111678589-b3dc1800-87dd-11eb-8258-a339b336fce4.png)
console error log, other than normal chrome caching errors, everything looks fine

**Comment by midnight-dev** [comment]
Out of the two systems with different behaviors, would you consider one of them faster than the other (in processing or network speed)? This sounds familiar to a strange race fault that made the vertex shader _consistently_ clip out virtually everything under the right conditions, but only on some systems.

**Comment by tjhanson** [comment]
no, both pc's have similar specs and internet connections. The computer that wont load the point cloud will load this point cloud that was also processed with 2.0 converter. 
https://fullcirclethinking.com/barringtonplaza/garageSL1.html
The only difference between the two is that this file loads 9 different point clouds

**Comment by tjhanson** [comment]
I also testing on a computer without dedicated graphics and minimal specs and everything loads fine, just takes a few more seconds

**Comment by midnight-dev** [comment]
Just for giggles, can you apply the change in #909 to your JavaScript to see if it resolves this abnormality?

**Comment by John-Hiroshi** [comment]
I have the same issue as well

**Comment by midnight-dev** [comment]
@John-Hiroshi Have you tried using the latest version from the `develop` branch? Or are you using the release 1.7.x or 1.8?

If you're absolutely using the latest commit, then there may be an issue with the current codebase (or a browser incompatibility). If you're using an earlier release, one of the later commits _might have_ fixed it.

By the way, can you copy and paste your console log output just so there's more information to attach to this?

**Comment by John-Hiroshi** [comment]
@midnight-dev I'M using 1.79 now.
Is the cause from `gps-time` ?  #909 
How do I fix this problem with code fixing but not changing the version?

I will give you the error information later.

**Comment by midnight-dev** [comment]
It's possible. Right now, the only (known) Converter 2.0 issue with 1.7 that behaves differently on different computers is the gps-time attribute bug. I can make a recommendation on what code you should change, but I'd need to know if you're okay with compiling from source or if you only want to modify the large, already-compiled _potree.js_ on the server. 

In the meantime, you can test to see if it's the GPS time by removing it from the _metadata.json_ file. That would prevent the shader from clipping out the points by this attribute. In theory, another quick and dirty workaround is to change the min & max values to something unequal (like 0 & 1 respectively) in that file. The block should be pretty obvious, though I can point it out if it's not. If it's being hosted somewhere behind a CloudFront cache, remember to invalidate the file from the cache so the edited version is served to potree.

**Comment by John-Hiroshi** [comment]
Thanks for your advice.

I got this error after removing the GPS time from the metadata.json file. Did I do the wrong way?

![image](https://user-images.githubusercontent.com/77081747/120576548-ebae3f80-c45d-11eb-902a-8d0285497838.png)

I tried the other workaround and it looks the issue is gone? 
![image](https://user-images.githubusercontent.com/77081747/120577183-ec93a100-c45e-11eb-961a-da8f74004cf0.png)

**Comment by John-Hiroshi** [comment]
Can I just change the material to not use the gps-time?
How do I do it? Probably at this code right?

`		Potree.loadPointCloud(url, "buggy point cloud", function(e){
			viewer.scene.addPointCloud(e.pointcloud);
			
			let material = e.pointcloud.material;
			material.size = 1;
			material.pointSizeType = Potree.PointSizeType.ADAPTIVE;
			
			e.pointcloud.position.x += 3;
			e.pointcloud.position.y -= 3;
			e.pointcloud.position.z += 4;
			
			viewer.fitToScreen();
		});`

**Comment by John-Hiroshi** [comment]
> @John-Hiroshi Have you tried using the latest version from the `develop` branch? Or are you using the release 1.7.x or 1.8?
> 
> If you're absolutely using the latest commit, then there may be an issue with the current codebase (or a browser incompatibility). If you're using an earlier release, one of the later commits _might have_ fixed it.
> 
> By the way, can you copy and paste your console log output just so there's more information to attach to this?

What specific version could fix this issue? Is it the latest version from the `develop` branch?

**Comment by midnight-dev** [comment]
Not sure regarding the new errors, and no, you cannot just change the material to fix it. 

The latest version in the default branch has merged in commits to address this unexpected behavior. I don't think the 1.8 release has it, so go with the most recent by cloning the repo and building potree by source.
